parser_test.go

```diff
@@ -311,13 +312,13 @@ func displaySliceDiff[T any](got []T, want []T) string {
 	maxLen := 0
 	for _, g := range got {
 		gs := fmt.Sprintf("%v", g)
-		maxLen = intMax(maxLen+1, len(gs))
+		maxLen = slices.Max([]int{maxLen + 1, len(gs)})
```

Comment:
There are min/max builtins now! You can just change this to:

```
maxLen = max(maxLen+1, len(gs))
```

And so on

Reply:
Oh, terrific! I was hesitant with slices.Max because the syntax looked cumbersome, and went down a bit of rabbit hole as to why it wasn't variadic.